language/move-bytecode-verifier/transactional-tests/tests/check_bounds/256_locals.mvir

```diff
@@ -1,4 +1,5 @@
-module {{default}}.M {
+//# publish
+module 0x0.M {
```

Comment:
Should we be using 0x0? Don't we restrict the usage of that?

Reply:
> Don't we restrict the usage of that?
It doesn't seem to be the case to me since everything runs fine. Maybe this was the case a long time ago, but probably no longer true at this time.

> Should we be using 0x0?
I don't have an answer now but I think we should think about it. At mean while, I'd be more than willing to change the existing usages to 0x2 or something else.